apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -367,6 +367,15 @@ export const getMultipleChoiceSurveyResults = (state) => {
   });
 };
 
+// Returns a boolean.  The selector function checks if the current assessment Id
+// is in the surveys structure.
+export const currentAssessmentIsSurvey = (state) => {
+  const surveysStructure = state.sectionAssessments.surveysByScript[state.scriptSelection.scriptId] || {};
```

Comment:
this line is getting a little hard to read. maybe break it up into:
```
const scriptId = state.scriptSelection.scriptId;
const surveysStructure = state.sectionAssessments.surveysByScript[scriptId] || {};
```

Reply:
on second thought, could `surveysStructure` just be `surveys`? i don't know that "structure" adds anything to my understanding of it, and it might make line 376 clearer (`Object.keys(surveys).includes(currentAssessmentId.toString());`)

Reply:
@madelynkasula I created another variable for scriptId per your feedback.  surveysStructure is used in multiple areas in the code and I will create another PR to address this.